.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: build
+on: push
+jobs:
+  tox:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        tox:
+          - py27
+          - py36
```

Comment:
if you only care about switching on python version, use that as your variable and then run `tox -e py` to use the interpreter for that version